bedrock/firefox/redirects.py

```diff
@@ -499,9 +496,6 @@ def firefox_mobile_faq(request, *args, **kwargs):
     # bug 1260423
     redirect(r'^firefox/choose/?$', 'firefox.new'),
 
-    # bug 1273837
-    redirect(r'^firefox(?:\/\d+\.\d+(?:\.\d+)?(?:a\d+)?)?/hello/start/?$', 'firefox.hello'),
```

Comment:
Oops, it seemed like this was caught by the `firefox/hello` redirect and was redundant, but I must have done something wrong when I tested. I've restored this redirect and grouped it with the others.